heroic-core/build.gradle

```diff
@@ -44,16 +45,20 @@ dependencies {
     implementation 'com.lightstep.tracer:tracer-okhttp'
     implementation 'com.lightstep.tracer:java-common'
     implementation 'com.lightstep.opencensus:lightstep-opencensus-exporter'
+
     implementation 'com.typesafe:config:1.3.2'
     implementation 'net.jodah:expiringmap:0.5.1'
+
+    // Default usage tracking module. Normally different module dependencies are handled in
+    // heroic-dist, but defaults need to be accessible in heroic-core.
+    implementation project(':heroic-usagetracking-google')
```

Comment:
nit; can we call this `heroic-usage-tracking-google-analytics' or `heroic-usage-tracking-google'?

Reply:
Sure